HoldingObjectsLab/PhoneBookRefactored/src/test/java/EntryTest.java

```diff
@@ -0,0 +1,33 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+
+/**
+ * Created by eugenevendensky on 2/7/17.
+ */
+public class EntryTest {
+
+
+    Entry entry;
+
+    @Before
+    public void setUp(){entry = new Entry("215-676-1408");}
+    @Test
+    public void addNumberTest(){
+        entry.addPhoneNumber("215-676-2222");
+        String actual = entry.getNumber(1);
+        String expected = "215-676-2222";
```

Comment:
frequently repeated string literal -- extract this to a constant so that if you choose to change it in one place it changes everywhere. [Examples](http://xunitpatterns.com/Literal%20Value.html)